src/main/docker/jenkins.swift.override.yml

```diff
@@ -0,0 +1,4 @@
+version: "2"
+services:
+    bamboo:
+        image: ghcr.io/ls1intum/artemis-jenkins-swift:latest
```

Comment:
I am a bit confused by this line: Why do you specify a `jenkins` image for the `bamboo` service?

Reply:
Good catch – this was just a copy-paste error. Will fix this!